#!/bin/sh -e
## compiler_defaults.dpatch by Eduard Bloch <blade@debian.org>
##
## DP: Be less verbose in non-debug builds

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
Index: a/src/yimage.cc
===================================================================
RCS file: /cvsroot/icewm/icewm-1.2/src/yimage.cc,v
retrieving revision 1.8
diff -u -r1.8 yimage.cc
--- a/src/yimage.cc	21 Mar 2004 13:53:04 -0000	1.8
+++ b/src/yimage.cc	13 Apr 2004 19:50:49 -0000
@@ -72,7 +72,7 @@
         fPixmap = fMask = None;
         fWidth = fHeight = 16;
     }
-    msg("%s %d %d", filename, fWidth, fHeight);
+    MSG(("%s %d %d", filename, fWidth, fHeight));
 #elif defined(CONFIG_XPM)
     XpmAttributes xpmAttributes;
     memset(&xpmAttributes, 0, sizeof(xpmAttributes));
Index: a/src/ypixbuf.cc
===================================================================
RCS file: /cvsroot/icewm/icewm-1.2/src/ypixbuf.cc,v
retrieving revision 1.22
diff -u -r1.22 ypixbuf.cc
--- a/src/ypixbuf.cc	6 Apr 2004 19:11:30 -0000	1.22
+++ b/src/ypixbuf.cc	13 Apr 2004 19:50:49 -0000
@@ -1009,7 +1009,7 @@
 
     if (fullAlpha)
         allocAlphaChannel();
-    msg("%s %d %d", filename, width(), height());
+    MSG(("%s %d %d", filename, width(), height()));
 }
 
 #if 0
